etna/authors/blocks.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+
+class AuthorBlock(blocks.StructBlock):
```

Comment:
Have you looked at using a separate model for authors so editors don't need to type an author's details each time?

Reply:
Good idea

Reply:
Thanks for adding the `Author` snippet!